.github/actions/workflow-run-job-linux/action.yml

```diff
@@ -182,3 +195,11 @@ runs:
         name: jobs-${{inputs.id}}
         path: jobs
         compression-level: 0
+
+    - name: Upload wheelhouse
+      if: always()
+      uses: actions/upload-artifact@v4
+      with:
+        name: wheelhouse-${{inputs.id}}
```

Comment:
I may refactor this in a followup to be more generic, but this is fine for now 👍